Doc updates for changes done in ConsentValidation Endpoint (#480)
wso2/docs-open-banking: en/docs/develop/consent-management-validate.md

| Name      | Type                  | Description  |
| ----------|-----------------------| -------------|
| headers   | JSONObject            | The request headers sent in the request. |
| body   | JSONObject            | The payload sent in the request. |

> | electedResource   | String            | The path invoked in the request. `ex: /accounts/{AccountId}`|

Reviewer: ```suggestion
| electedResource   | String            | The path invoked in the request. For example: `/accounts/{AccountId}`|
```

| consentId | String                | The consent ID related to the current resource request. This consent ID is bound to the user access token used. |
| userId | String                | The user ID related to the current resource request. This user ID is retrieved from the user access token. |
| comprehensiveConsent  | DetailedConsentResource   | The comprehensive consent object related to the consent that is being validated. This object contains all the details related to the consent. |

> | resourceParams | Map<String, String>                | A map containing the full resource path with query parameters `(ex: aisp/accounts/{AccountId}?queryParam=urlEncodedQueryParamValue)`, HTTP method, and context `(ex: /open-banking/v3.1/aisp)` of the request. |

Reviewer: ```suggestion
| resourceParams | Map<String, String>                | A map containing the full resource path with query parameters (ex: `aisp/accounts/{AccountId}?queryParam=urlEncodedQueryParamValue`), HTTP method, and context (ex: `/open-banking/v3.1/aisp`) of the request. |
```

              "Accept":"application\/json",
              "Content-Type":"application\/json; charset=UTF-8"
           },
           "body":{
                "Data": {
                    "Permissions": [
                        "ReadAccountsBasic",
                        "ReadAccountsDetail",
                        "ReadBalances"
                    ],
                    "ExpirationDateTime": "2021-08-21T12:55:29.279+05:30",
                    "TransactionFromDateTime": "2021-08-16T12:55:29.331+05:30",
                    "TransactionToDateTime": "2021-08-19T12:55:29.331+05:30"
                },
                "Risk": {

                }
            },
           "consentId":"5f4e0ded-4768-4bd3-adb9-7a9587415b05",
           "resourceParams":{
              "resource":"\/aisp\/accounts?fromDateTime=2021-05-12T12%3A24%3A50.799%2B05%3A30&toDateTime=2021-05-12T12%3A24%3A50.799%2B05%3A30",
              "context":"\/open-banking\/v3.1\/aisp",
              "httpMethod":"POST"
           },
           "userId":"admin@wso2.com@carbon.super@carbon.super",
           "electedResource":"\/accounts"

>        }

Reviewer: ```suggestion
        }
```
